packages/ui/src/components/Visualization/ContextToolbar/RuntimeSelector/RuntimeSelector.tsx

```diff
@@ -21,7 +21,7 @@ const getIcon = (name: string) => {
         <img src={quarkusLogo} />
       </Icon>
     );
-  } else if (name.includes('SpringBoot')) {
+  } else if (name.includes('Spring Boot')) {
```

Comment:
@tplevko maybe we could say something similar to: this way it should work for both old catalog and the new one

```typescript
name.replace(' ', '').includes('SpringBoot')
```

Reply:
thanks for the hint - updated